src/current/v25.1/set-up-logical-data-replication.md

```diff
@@ -34,31 +40,36 @@ You'll need:
     - All nodes in each cluster will need access to the Certificate Authority for the other cluster. Refer to [Step 2. Connect from the destination to the source](#step-2-connect-from-the-destination-to-the-source).
-- LDR replicates at the table level, which means clusters can contain other tables that are not part of the LDR job. If both clusters are empty, create the tables that you need to replicate with **identical** schema definitions (excluding indexes) on both clusters. If one cluster already has an existing table that you'll replicate, ensure the other cluster's table definition matches. For more details on the supported schemas, refer to [Schema Validation](#schema-validation).
-
-{% comment  %}To add later, after further dev work{{site.data.alerts.callout_info}}
-If you need to run LDR through a load balancer, use the load balancer IP address as the SQL advertise address on each cluster. It is important to note that using a load balancer with LDR can impair performance.
-{{site.data.alerts.end}}{% endcomment %}
-
-To create bidirectional LDR, you can complete the [optional step](#step-4-optional-set-up-bidirectional-ldr) to start the second LDR job that sends writes from the table on cluster B to the table on cluster A.
+- LDR replicates at the table level, which means clusters can contain other tables that are not part of the LDR job. (For [`CREATE LOGICAL REPLICATION STREAM`](#create-logical-replication-stream) only): If both clusters are empty, create the tables that you need to replicate with **identical** schema definitions (excluding indexes) on both clusters. If one cluster already has an existing table that you'll replicate, ensure the other cluster's table definition matches. For more details on the supported schemas, refer to [Schema Validation](#schema-validation).
 
 ### Schema validation
 
 Before you start LDR, you must ensure that all column names, types, constraints, and unique indexes on the destination table match with the source table.
 
-You cannot use LDR on a table with a schema that contains the following:
+You cannot use LDR on a table with a schema that contains:
 
 - [Column families]({% link {{ page.version.version }}/column-families.md %})
 - [Partial indexes]({% link {{ page.version.version }}/partial-indexes.md %}) and [hash-sharded indexes]({% link {{ page.version.version }}/hash-sharded-indexes.md %})
 - Indexes with a [virtual computed column]({% link {{ page.version.version }}/computed-columns.md %})
 - Composite types in the [primary key]({% link {{ page.version.version }}/primary-key.md %})
 
+Additionally, for the `CREATE LOGICALLY REPLCATED` syntax, you cannot use LDR on a table with a schema that contains:
```

Comment:
i find it a bit confusing that we have this list of limitations as well as a separate list of known limitations. LDR, with both syntaxes, currently do not support UDFs, Sequences etc. IMHO, it would be clearer to have just one list.

Reply:
For this, I've created myself an issue that I'll tackle as a follow-up. I want to take a look at the KL sections across LDR and this Schema validation section to evaluate the best approach: https://cockroachlabs.atlassian.net/browse/DOC-12331